qemu/tests/timedrift_with_stop.py

```diff
@@ -4,10 +4,12 @@
 import signal
 
 from autotest.client.shared import error
+from autotest.client import utils
 
 from virttest import utils_test
 
 
+@error.context_aware
```

Comment:
`error`  is not a suggested module for new PR,  use error_context in avocado-vt instead. such as:
remove  from autotest.client.shared import error
add   from virttest import error_context

@error_context.context_aware

Reply:
Thanks. Updated.